proposals/infra/INF-0004-binding-resource-binding-diagnostics.md

```diff
@@ -0,0 +1,212 @@
+* Proposal: [0004](0004-binding-prefixes.md)

```

Comment:
Should't this link to this file?  The name doesn't seem to match.

(Note also that "binding" appears twice in this file's current name.

Reply:
Fixed